common/api/core-bentley.api.md

```diff
@@ -222,6 +222,12 @@ export enum ChangeSetStatus {
     WrongDgnDb = 90130
 }
 
+// @internal
```

Comment:
What's the reason for marking `@internal`?

Reply:
to prevent external consumers from depending on it, it's not really designed for external use at the moment, although it could be vetted for external use, it's not very far from it. If you think it would be useful to provide to our users then I could check through it and make it non external. Maybe @ColinKerr has a comment since he thought it should be internal

Reply:
nah that seems fine.